src/styles/color.stories.tsx

```diff
@@ -0,0 +1,116 @@
+      </div>
+      <div css={HeadingCss}>Text & Icon</div>
+      <div css={FlexBoxCss}>
+        <div css={[(theme) => ({ backgroundColor: theme.colors.secondary_200 }), BoxCss]} />
+        <div css={[(theme) => ({ backgroundColor: theme.colors.secondary_100 }), BoxCss]} />
+        <div css={[(theme) => ({ backgroundColor: theme.colors.secondary_300 }), BoxCss]} />
+      </div>
+      <div css={FlexBoxCss}>
+        <span>$secondary.200</span>
+        <span>$secondary.100 (button-disabled)</span>
+        <span>$secondary.300 (button-hover)</span>
+      </div>
+      <div css={HeadingCss}>Background</div>
+      <div css={FlexBoxCss}>
+        <div css={[(theme) => ({ backgroundColor: theme.colors.black }), BoxCss]} />
+        <div css={[(theme) => ({ backgroundColor: theme.colors.gray_500 }), BoxCss]} />
+        <div css={[(theme) => ({ backgroundColor: theme.colors.gray_400 }), BoxCss]} />
+        <div css={[(theme) => ({ backgroundColor: theme.colors.gray_300 }), BoxCss]} />
+      </div>
+      <div css={FlexBoxCss}>
+        <span>$black (text-primary)</span>
+        <span>$gray.500 (text-secondary)</span>
+        <span>$gray.400 (text-tertiary)</span>
+        <span>$gray.300 (text-disabled)</span>
+      </div>
+      <div css={FlexBoxCss}>
+        <div css={[(theme) => ({ backgroundColor: theme.colors.white }), BoxCss]} />
+        <div css={[(theme) => ({ backgroundColor: theme.colors.gray_50 }), BoxCss]} />
+        <div css={[(theme) => ({ backgroundColor: theme.colors.gray_100 }), BoxCss]} />
+        <div css={[(theme) => ({ backgroundColor: theme.colors.gray_200 }), BoxCss]} />
+      </div>
+      <div css={FlexBoxCss}>
+        <span>$white (background-primary)</span>
+        <span>$gray.50 (background-secondary)</span>
+        <span>$gray.100 (background-hover)</span>
+        <span>$gray.200 (background-tertiary)</span>
+      </div>
+      <div css={HeadingCss}>System Color</div>
+      <div css={FlexBoxCss}>
+        <div css={[(theme) => ({ backgroundColor: theme.colors.red }), BoxCss]} />
+      </div>
+      <div css={FlexBoxCss}>
+        <span>$system-red</span>
+      </div>
+      <div css={HeadingCss}>Sub Color</div>
+      <div css={FlexBoxCss}>
+        <div css={[(theme) => ({ backgroundColor: theme.colors.pink }), BoxCss]} />
+        <div css={[(theme) => ({ backgroundColor: theme.colors.purple }), BoxCss]} />
+        <div css={[(theme) => ({ backgroundColor: theme.colors.yellowgreen }), BoxCss]} />
+        <div css={[(theme) => ({ backgroundColor: theme.colors.bluegreen }), BoxCss]} />
+        <div css={[(theme) => ({ backgroundColor: theme.colors.skyblue }), BoxCss]} />
+      </div>
+      <div css={FlexBoxCss}>
+        <span>$sub-pink</span>
+        <span>$sub-purple</span>
+        <span>$secondary-yellowgreen</span>
+        <span>$secondary-bluegreen</span>
+        <span>$secondary-skyblue</span>
+      </div>
+    </ThemeProvider>
+  );
+}
+
+const HeadingCss = css`
```

Comment:
```suggestion
const headingCss = css`
```

해당 파일에서는 수정이 안되어도 괜찮을 거 같은데, 다른 부분에서는 이렇게 소문자로 시작하면 좋을 거 같아요 !! 👍 👍